components/LandingPage.js

```diff
@@ -46,9 +46,9 @@ export default class LandingPage extends Component {
         </View>
         {this.renderImage()}
         <View style={s.box}>
-          {intro.length ? <Text style={{textAlign:'center',fontSize:25}}>{intro}</Text> : null}
-          {title.length ? <Text style={{textAlign:'center',fontSize:25}}>{title}</Text> : null}
-          {des.length ? <Text style={{textAlign:'center',fontSize:16,padding:20}}>{des}</Text> : null}
+          {intro.length ? <Text style={{textAlign:'center',fontSize:25, color: '#364247'}}>{intro}</Text> : null}
```

Comment:
make this color a `const` near the top of the file (or maybe even import it from a common file, since it is used in multiple locations)